src/itype.h

```diff
@@ -623,6 +625,11 @@ struct islot_battery {
     units::energy max_capacity;
 };
 
+struct islot_air {
+    /** Maximum energy the battery can store */
+    units::energy max_capacity;
```

Comment:
These are not energy units. And the comment above should mention air, not the battery and not the energy.

Reply:
I changed the comment, but in this case the compressed air is energy, as it is being used like stored energy, not for breathing. See compressed air energy storage.

Reply:
Don't you declare here maximum _volume_ of air cylinders, and not the compressed air energy storage value?